packs/st2/actions/executions_list.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: "executions.list"
+enabled: true
+description: "Retrieve a list of executions."
+runner_type: run-python
+entry_point: executions_list.py
+parameters:
+  action:
+    type: "string"
+    description: "Optional action to filter the executions on"
+    required: false
+  status:
```

Comment:
Yeah, I was thinking about it.

I think in this case it's fine to push a bit work to the API - this way it will be less maintenance and code which needs to be updated if we introduce a new status (which, I guess, it's unlikely at this point, but who knows :D)